service/src/main/java/com/codeforcommunity/requester/MapRequester.java

```diff
@@ -242,6 +244,16 @@ public void updateBlocks(List<String> blockIds, BlockStatus updateTo) {
         });
   }
 
+  public Object getPrivateStreets() {
```

Comment:
Small nitpick - move this to below the `public updateBlocks()` method to keep public methods together #OOD :)